src/root.component.tsx

```diff
@@ -3,11 +3,21 @@ import { Route } from "react-router";
 import { BrowserRouter } from "react-router-dom";
 import openmrsRootDecorator from "@openmrs/react-root-decorator";
 import PatientChartSummary from "./summary/patient-chart-summary.component";
+import PatientChartLevelTwo from "./summary/patient-chart-level-two.component";
 
 function Root(props) {
   return (
-    <BrowserRouter>
-      <Route to="/patient/:patientUuid/chart" component={PatientChartSummary} />
+    <BrowserRouter basename={window["getOpenmrsSpaBase"]()}>
+      <Route
```

Comment:
I think we could set up the routes a bit better than this, so that we don't unmount/remount the patient banner every time you go between level one and level two of the patient chart (which is what your code is currently doing).

```tsx
<BrowserRouter>
  <Route to="/patient/:patientUuid/chart" render={routeProps => (
     <PatientBanner
        match={props.match}
        patient={currentPatient}
        showPatientSummary={setShowPatientSummary}
      />
  )} />
  <Route to="/patient/:patientUuid/chart" exact component={PatientChartSummary} />
  <Route to="/patient/:patientUuid/chart/:chartSlug" exact component={PatientChartLevelTwo} />
</BrowserRouter>
```